src/Etl/Dto/FilterFieldModelDto.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Sherl\Sdk\Etl\Dto;
+
+use JMS\Serializer\Annotation as Serializer;
+
+class FilterFieldModel
```

Comment:
```suggestion
class FilterFieldModelDto
```

Il faudrait que le nommage de la classe corresponde au nom du fichier pour que ce soit un peu plus simple de s'y retrouver.